packages/cli/src/services/check-runner.ts

```diff
@@ -91,7 +91,7 @@ export default class CheckRunner extends EventEmitter {
       await checksApi.run(checkRun)
     } catch (err: any) {
       if (err?.response?.status === 402) {
-        const errorMessage = `Failed to run a check. ${err.response.data.message}`
+        const errorMessage = `Failed to run a check. ${err.message}`
```

Comment:
I think that we can leave the `err.response.data.message` in this case. I think that it should be fine since we're already checking that the error has a specific `402` status code.